markdown/en/documentation/tracking-guide.md

```diff
@@ -6,13 +6,13 @@ This guide covers the integration of all necessary events.
 ### Click Tracking
 
 ---
-The Click-Tracking for products, recommendations and campaigns is integrated in the same way.
+The integration of Click Tracking for products, recommendations and campaigns is the same.
 
-The `ff-record-list` element detects automatically if the product which are displayed belong to campaigns, recommendations or the search result. So their is no need to distinguish between recommendation click, campaign click or common product click.
+The `ff-record-list` element detects automatically if the displayed products belong to campaigns, recommendations or the search result. So there is no need to distinguish between recommendation click, campaign click or common product click.
 
 
 #### Tracking with `data-redirect`
-The `[data-redirect]` attribute redirects the user on click to the product detail page. While without using the `[data-redirect]` attribute the browser may reloads the page to fast for the tracking request to succeed we added the tracking behavior to this directive.
+The `[data-redirect]` attribute redirects the user on click to the product detail page. While without using the `[data-redirect]` attribute the browser may reload the page too fast for the tracking request to succeed, we added the tracking behavior to this directive.
```

Comment:
_While without [...], we added_ Is that good english? I find it hard to read

Reply:
What about:
_We added the tracking behavior to this directive as page reloads may be too fast for tracking requests to succeed when not using the `[data-redirect]` attribute._

Does this still have the same meaning?

Reply:
I think so